nexus/posts/models.py

```diff
@@ -0,0 +1,33 @@
+# Third Party Stuff
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+# Nexus Stuff
+from nexus.base.models import TimeStampedUUIDModel
+
+
+class Post(TimeStampedUUIDModel):
+    """ Post model class to provide post related field
+    """
+    posted_by = models.ForeignKey(
+        'users.User',
```

Comment:
So much indentation is not needed here. Please fix this.

Reply:
ACK